src/v0/destinations/salesforce/transform.js

```diff
@@ -84,12 +87,18 @@ function responseBuilderSimple(
   }
 
   const response = defaultRequestConfig();
-  const header = {
+  const finalHeader = authorizationFlow === 'oauth' ? 
+  { 
     'Content-Type': JSON_MIME_TYPE,
-    Authorization: authorizationData.token,
+    Authorization: `Bearer ${authorizationData.token}`
+   } : 
+   {
+    'Content-Type': JSON_MIME_TYPE,
+    Authorization: authorizationData.token
```

Comment:
Why don't we need Bearer for non oauth type?

Reply:
this is how the original implementation was, the access token was generated differently.